From 59ed3e02eb12f364578a6920ef40b554cff6f415 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Thu, 17 May 2018 11:49:07 +0800
Subject: [PATCH] phy: rockchip-inno-usb2: fix compile warning when !PM_SLEEP

When CONFIG_PM_SLEEP is disabled, we get a warning about
unused functions:

drivers/phy/rockchip/phy-rockchip-inno-usb2.c:1828:12: warning:
'rockchip_usb2phy_low_power_enable' defined but not used

Marking it as __maybe_unused avoids the warning without introducing
an ugly #ifdef.

Change-Id: I644b00eb2b30d9e4b63da46b7f387a7571b0f103
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-inno-usb2.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-inno-usb2.c b/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
index 66fb407962cc..fe28fe92be91 100644
--- a/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
+++ b/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
@@ -1825,9 +1825,10 @@ static int rockchip_usb2phy_probe(struct platform_device *pdev)
 	return ret;
 }
 
-static int rockchip_usb2phy_low_power_enable(struct rockchip_usb2phy *rphy,
-					struct rockchip_usb2phy_port *rport,
-					bool value)
+static int __maybe_unused
+rockchip_usb2phy_low_power_enable(struct rockchip_usb2phy *rphy,
+				  struct rockchip_usb2phy_port *rport,
+				  bool value)
 {
 	int ret = 0;
 
-- 
2.35.3

